### ISSUE TITLE
Support operator module callables in inspect.signature

### ISSUE THREAD

**Issue by a91f67b1-65f1-452b-975e-bf8c64c6df62** [python/cpython#86951]
BPO | [42785](https://bugs.python.org/issue42785)
--- | :---
Nosy | @plammens

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2020-12-30.12:38:08.138>
labels = ['type-feature', 'library', '3.9']
title = 'Support operator module callables in inspect.signature'
updated_at = <Date 2020-12-30.12:39:03.612>
user = 'https://github.com/plammens'
```

bugs.python.org fields:
```python
activity = <Date 2020-12-30.12:39:03.612>
actor = 'plammens'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2020-12-30.12:38:08.138>
creator = 'plammens'
dependencies = []
files = []
hgrepos = []
issue_num = 42785
keywords = []
message_count = 2.0
messages = ['384061', '384062']
nosy_count = 1.0
nosy_names = ['plammens']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue42785'
versions = ['Python 3.9']
```

</p></details>

**Comment by wRAR** [comment]
Looks like this is fixed in Python 3.12:

```
>>> import inspect
>>> import operator
>>> inspect.signature(operator.attrgetter("spam"))
<Signature (*args, **kwargs)>
```

**Comment by wRAR** [comment]
Specifically, it's fixed in the unreleased 3.12.3 (currently available in Debian unstable), probably by #116198
